internal/service/docdb/sweep.go

```diff
@@ -23,6 +23,212 @@ func init() {
 			// "aws_docdb_cluster",
 		},
 	})
+	resource.AddTestSweepers("aws_docdb_subnet_group", &resource.Sweeper{
+		Name:         "aws_docdb_subnet_group",
+		F:            sweepDBSubnetGroups,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_event_subscription", &resource.Sweeper{
+		Name:         "aws_docdb_event_subscription",
+		F:            sweepEventSubscriptions,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster", &resource.Sweeper{
+		Name:         "aws_docdb_cluster",
+		F:            sweepDBClusters,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster_snapshot", &resource.Sweeper{
+		Name:         "aws_docdb_cluster_snapshot",
+		F:            sweepDBClusterSnapshots,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster_instance", &resource.Sweeper{
+		Name:         "aws_docdb_cluster_instance",
+		F:            sweepDBInstances,
+		Dependencies: []string{},
+	})
+	resource.AddTestSweepers("aws_docdb_cluster_parameter_group", &resource.Sweeper{
+		Name:         "aws_docdb_cluster_parameter_group",
+		F:            sweepDBClusterParameterGroups,
+		Dependencies: []string{},
```

Comment:
If there are any instances using the parameter group, you'll get the error

> InvalidDBParameterGroupState: One or more database instances are still members of this parameter group test, so the group cannot be deleted

You'll need to add a dependency, i.e.,
```suggestion
		Dependencies: []string{
			"aws_docdb_cluster_instance",
		},
```